readinglist/utils.py

```diff
@@ -4,11 +4,16 @@
     import json  # NOQA
 
 import ast
+import time
+
 from colander import null
 
+
 # removes whitespace, newlines, and tabs from the beginning/end of a string
 strip_whitespace = lambda v: v.strip(' \t\n\r') if v is not null else v
```

Comment:
This is used as a *preparer* for a Colander node, which gives ``colander.null`` when no value is provided...
We could put ``not in (null, None)`` to extend its usage outside colander indeed.